CODEOWNERS

```diff
@@ -24,7 +20,7 @@
 /src/elements/content-preview/              @box/preview
 /src/elements/content-sharing/              @box/partners
 /src/elements/content-sidebar/              @box/preview
-/src/elements/content-uploader/             @box/create-communicate
+/src/elements/content-uploader/             @box/preview
```

Comment:
Let's revert this for now, since `@box/preview` is already a child team under `@box/create-communicate`.

Reply:
reverted